blocks/serializer/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import * as query from 'hpq';
+
+/**
+ * Takes a block list and returns the serialized post content
+ *
+ * @param  {Array}  blocks Block list
+ * @return {String}        The post content
+ */
+export default function serialize( blocks ) {
+	return blocks.reduce( ( memo, block ) => {
```

Comment:
Yes, the difficulty here is to figure out which attributes needs to be serialized to the starting comment and which should be ignored (because they're already in the static content).

So what I'm doing is running the `attributes` object (or function) defined on each block just to know which "keys" (attributes) are extracted from the content.